frontend/packages/process-editor/src/components/ConfigPanel/ConfigContent/ConfigContent.tsx

```diff
@@ -14,6 +14,7 @@ import { EditPolicy } from './EditPolicy';
 import { EditDataTypesToSign } from '../EditDataTypesToSign';
 import { EditUniqueFromSignaturesInDataTypes } from '../EditUniqueFromSignaturesInDataTypes';
 import { StudioModeler } from '../../../utils/bpmnModeler/StudioModeler';
+import { NewNameRecommendation } from '@altinn/process-editor/components/ConfigPanel/ConfigContent/EditTaskRecommendedItems/NewNameRecommendation';
```

Comment:
Would it be simpler to do this? 😊 

```suggestion
import { NewNameRecommendation } from './EditTaskRecommendedItems/NewNameRecommendation';
```